tsconfig.json

```diff
@@ -1,7 +1,9 @@
 {
   "include": [
     "core/**/*",  // N.B.: also pulls in closure/goog/goog.js if needed.
```

Comment:
Why does this one pull in goog.js if needed, but the other closure files need to be included explicitly?

Reply:
Because apparently `closure-make-deps` will follow `import` statements.  (It can't do that for `goog.require`s, as they don't have filepaths.)  It might be, actually, that we could just include `core/blockly.ts` and that would be enough…

Reply:
> Because apparently `closure-make-deps` will follow `import` statements. (It can't do that for `goog.require`s, as they don't have filepaths.) It might be, actually, that we could just include `core/blockly.ts` and that would be enough…

Sorry, this is totally wrong because brain was not engaged.  The correct answer is:

Because `tsc` will follow `import` statements.

And it _is_ sufficient to include only `blockly.ts`, `any_aliases.ts` and `main.js` (the three files that are not imported by any other file), but I think it is a little weird to do that.